packages/gatsby/src/schema/build-connection-fields.js

```diff
@@ -104,7 +104,30 @@ module.exports = type => {
           groupConnections.push(groupConn)
         })
 
-        return groupConnections
+        // make tags arrays case ignored
+        const tags = groupConnections.map(e=>{
```

Comment:
I don't think this is the best spot for this logic. The helpers here are for generically constructing infered types, and having very specific markdown logic here may lead to stuff getting out of sync or ignored in the future. Ideally you'd want to locate this with the code that produces the data, e.g. the transformer-remark plugin. and make use of gatsby's `extendNode` api to tweak the creation of the tags